.github/workflows/shiplog-sign.yml

```
@@ -0,0 +1,97 @@
+        run: |
+          set -euo pipefail
+          git shiplog init
+          git shiplog --boring --yes write
+
+      - name: Push Shiplog journal
+        env:
+          SHIPLOG_ENV: ${{ inputs.env || 'prod' }}
+        run: |
+          set -euo pipefail
+          git push origin "refs/_shiplog/journal/${SHIPLOG_ENV}"
+
```

YAMLlint (1.37.1)

[error] too many blank lines (1 > 0)

(empty-lines)